app/services/clients/zendesk.client.js

```diff
@@ -1,33 +1,42 @@
 'use strict'
 
-const zendesk = require('node-zendesk')
+const axios = require('axios')
 const zendeskConfig = require('../../../config/zendesk')
 
-const zendeskClient = zendesk.createClient({
-  username: process.env.ZENDESK_USER,
-  token: process.env.ZENDESK_API_KEY,
-  remoteUri: process.env.ZENDESK_URL,
-  proxy: process.env.http_proxy
-})
+async function createTicket (opts) {
+  try {
+    const zendeskTicketData = {
+      ticket: {
+        requester: {
+          email: opts.email,
+          name: opts.name
+        },
+        type: opts.type,
+        subject: opts.subject,
+        comment: {
+          body: opts.message
+        },
+        group_id: zendeskConfig.GROUP_ID,
+        organization_id: zendeskConfig.ORG_ID,
+        ticket_form_id: zendeskConfig.FORM_ID,
+        tags: opts.tags
+      }
+    }
 
-function createTicket (opts) {
-  return zendeskClient.tickets.create({
-    ticket: {
-      requester: {
-        email: opts.email,
-        name: opts.name
-      },
-      type: opts.type,
-      subject: opts.subject,
-      comment: {
-        body: opts.message
-      },
-      group_id: zendeskConfig.GROUP_ID,
-      organization_id: zendeskConfig.ORG_ID,
-      ticket_form_id: zendeskConfig.FORM_ID,
-      tags: opts.tags
+    const zendeskOptions = {
+      method: 'post',
+      url: process.env.ZENDESK_URL + '/tickets',
+      data: zendeskTicketData,
+      auth: {
+        username: process.env.ZENDESK_USER + '/token',
+        password: process.env.ZENDESK_API_KEY
+      }
     }
-  })
+
+    await axios(zendeskOptions)
```

Comment:
The previous method this is replacing returns the created zendesk ticket once it is created, this changes the functionality by returning nothing (I'm guessing the created ticket is then not used by the rest of the code, just something to be aware of)